src/network/providers/ethereum.js

```diff
@@ -196,7 +205,11 @@ ethereum.loadPointContract = async (
             log.debug({contractName, at}, `Abi is not found locally, compiling`);
 
             const contractPath = path.resolve(basepath, '..', 'hardhat', 'contracts');
-            await compileAndSaveContract({name: contractName, contractPath, CONTRACT_BUILD_DIR});
```

Comment:
The main culprit here - missing `buildDirPath` key

Reply:
Good catch, thanks @JatinBumbra!